layouts/shortcodes/op-datadog-archives-s3-setup.md

```diff
@@ -0,0 +1,25 @@
+1. Navigate to [Amazon S3][101]. Create an S3 bucket to send your archives to. Do not make your bucket publicly readable.
+
+2. Create a policy with the following permissions. Make sure to update the bucket name to the name of the S3 bucket you created earlier.
+
+    ```json
```

Comment:
@barieom  I updated Ari's version to what is in the [Log Archives doc](https://docs.datadoghq.com/logs/log_configuration/archives/?tab=awss3#set-permissions). Ari didn't have "s3:GetObject", which I believe is needed for rehydrating. Is that correct? Is there any reason the policy shouldn't be the same as what's in the Log Archives doc?

Reply:
yup you're correct, it should be identical to the Log Archives doc